src/pages/resultsView/plots/PlotsTab.tsx

```diff
@@ -2554,6 +2564,52 @@ export default class PlotsTab extends React.Component<IPlotsTabProps, {}> {
         },
     });
 
+    @computed get mutationDataForColoring() {
```

Comment:
not sure what about these guys is "forColoring".    maybe rename to describe what they are as opposed to how they are used?  and is there any way to generalize?  maybe not.  but like, there are other data types right?  might we ever want to color by them?

Reply:
this is the data that's loaded for annotation, i.e. the colors of the fill and stroke of the circles. I thought "ForColoring" makes sense because its related to ColoringMenuSelection